lib/util/tXml.js

```diff
@@ -815,6 +821,28 @@ shaka.util.TXml = class {
   }
 
 
+  /**
+   * Search the node index by the t attribute
+   * and return the index. if not found return null
+   * @param {!Array<shaka.extern.xml.Node>} nodes
+   * @param {!string} attribute
+   * @param {!number} value
+   * @private
+   */
+  static nodePositionByAttribute_(nodes, attribute, value) {
+    let index = 0;
+    for (const node of nodes) {
```

Comment:
could be `for(let i;...)` because you need to increment counter on your own anyway.

Reply:
let me do

Reply:
@tykus160 
it will be quite similar 
`for (let index = 0; index < nodes.length; index++) {
      const node = nodes[index];
      const attrs = node.attributes;
      const val = Number(attrs[attribute]);
      if (val === value) {
        return index;
      }
      index++;
    }`